arch/arm64/boot/dts/qcom/sdm660.dtsi

```diff
@@ -0,0 +1,378 @@
+// SPDX-License-Identifier: GPL-2.0-only
+/* Copyright (c) 2018, Craig Tatlor.
```

Comment:
I know this is used that way in some files, but personally this comment format is hurting my eyes :D

```suggestion
/*
 * Copyright (c) 2018, Craig Tatlor.
```

Reply:
:) :ok: